cloud-consumption/src/main/java/com/sequenceiq/consumption/configuration/api/CloudbreakApiConfig.java

```diff
@@ -25,8 +29,23 @@ public class CloudbreakApiConfig {
     @Value("${cert.ignorePreValidation:true}")
     private boolean ignorePreValidation;
 
+    @Inject
+    private RegionAwareInternalCrnGeneratorFactory regionAwareInternalCrnGeneratorFactory;
+
+    private CloudbreakServiceUserCrnClient cloudbreakClient() {
+        return new CloudbreakUserCrnClientBuilder(cloudbreakServerUrl)
+                .withCertificateValidation(false)
+                .withIgnorePreValidation(true)
+                .withDebug(true)
+                .build();
+    }
+
     @Bean
-    public CloudbreakApiClientParams cloudbreakClient() {
-        return new CloudbreakApiClientParams(restDebug, certificateValidation, ignorePreValidation, cloudbreakServerUrl);
+    public CloudbreakInternalCrnClient client() {
+        return new CloudbreakInternalCrnClient(cloudbreakClient(), internalCrnBuilder());
+    }
+
+    private RegionAwareInternalCrnGenerator internalCrnBuilder() {
+        return regionAwareInternalCrnGeneratorFactory.sdxAdmin();
```

Comment:
coreAdmin perhaps?

Reply:
Strictly speaking neither is perfect. These are legacy internal Altus service types. :smile: